src/components/Table/Table.styles.tsx

```
@@ -0,0 +1,37 @@
+
+import { StyledTableCellProps } from './Table.types';
+export const StyledHeader = styled(MuiTableHead)<MuiTableHeadProps>(
+    ({ theme: { spacing } }) => ({
+        backgroundColor: palette.gray[50],
+        '& th td': {
+            padding: spacing(4),
+        },
+    }),
+);
+
+export const StyledTableRow = styled(MuiTableRow)(({ theme: {} }) => ({
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)